deploy/docker/Dockerfile

```diff
@@ -1,27 +1,44 @@
 # Dockerfile for transtats
```

Comment:
we may keep this `Dockerfile` as is. README.md may contain info to build `Dockerfile with env vars`